libs/dmlf/tests/execution/basic_engine_tests.cpp

```diff
@@ -1442,7 +1442,7 @@ TEST(BasicVmEngineDmlfTests, AddFixed)
       "add", "state", "add", Params{LedgerVariant(fp64_t(4.5)), LedgerVariant(fp32_t(5.5))});
   EXPECT_TRUE(result.succeeded());
   // std::cout << result.error().message() << '\n';
```

Comment:
please ditch this commented out code while you're there

Reply:
fixed